web/src/engine/osk/src/input/gestures/browser/keytip.ts

```diff
@@ -4,11 +4,15 @@ import KeyTipInterface from '../../../keytip.interface.js';
 import VisualKeyboard from '../../../visualKeyboard.js';
 import { GesturePreviewHost } from '../../../keyboard-layout/gesturePreviewHost.js';
 
+const DEFAULT_TIP_ORIENTATION = 'up';
```

Comment:
I think we should have a 

```suggestion
type KeyTipOrientation = 'up' | 'down';
const DEFAULT_TIP_ORIENTATION: KeyTipOrientation = 'up';
```